specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/Builds.json

```diff
@@ -1,486 +0,0 @@
-{
```

Comment:
Why are these resource types being removed?

Reply:
Thanks @visingla-ms for your comments. These resource/resource types correspond to an API version that has not yet been released, and that we don't want to release anymore in the 2023-05-02-preview version.
As such, we're just removing these in this PR. In the future, we'll open a new PR once we know which version to target on our side. Again, this API version was never released with these changes in the first place.

Reply:
@konrad-jamrozik Tagging you as asked separately. These resource troupes are OK to remove (cf. above), and as such there are no more comments for us to address.